src/middleware/builtin.spec.ts

```diff
@@ -5,7 +5,7 @@ import sinon from 'sinon';
 import { ErrorCode } from '../errors';
 import { Override, delay, wrapToResolveOnFirstCall } from '../test-helpers';
```

Comment:
Looks like it's not being used as well

```suggestion
import { Override, delay } from '../test-helpers';
```